### PR TITLE
added explicit UTF-8 encoding when reading to byte

### PR DESCRIPTION
**Pull request by Geisterfalle**
## Changes

Added explicit conversion to UTF-8.

- First change

## Related issue

Fixes https://github.com/ehrbase/project_management/issues/112

## Additional information and checks

- [ ] Pull request linked in changelog

### CODE REVIEW

File: service/src/main/java/org/ehrbase/service/TemplateFileStorageService.java [hunk 1]
```diff
@@ -123,7 +121,7 @@ void setOptPath(String optPath) {
     public void storeTemplate(OPERATIONALTEMPLATE template) {
         XmlOptions opts = new XmlOptions();
         opts.setSaveSyntheticDocumentElement(new QName("http://schemas.openehr.org/v1", "template"));
-        saveTemplateFile(template.getTemplateId().getValue(), template.xmlText(opts).getBytes());
+            saveTemplateFile(template.getTemplateId().getValue(), template.xmlText(opts).getBytes(StandardCharsets.UTF_8));
```
Comment (ppazos) [inline service/src/main/java/org/ehrbase/service/TemplateFileStorageService.java]: @Geisterfalle please fix indentation

**Comment by Geisterfalle** [comment]
Thanks Pablo. You can use any template and when there are no JVM parameters or environment variables provided, the upload fails on system were UTF-8 is not standard encoding (windows, some Linux distributions).

**Comment by Geisterfalle** [comment]
@ppazos should be good now, please have a look

**Comment by ppazos** [comment]
@Geisterfalle I'm not running the system with an encoding parameter, in fact I never did, and didn't got errors when uploading templates on my environment, so I can't reproduce since I don't have the same env.

**Pull request merged**